optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/roster/RosterGenerator.java

```diff
@@ -578,9 +583,10 @@ public RosterState createRosterState(GeneratorType generatorType, Tenant tenant,
             String name = employeeNameGenerator.generateNextValue();
             HashSet<Skill> skillProficiencySet = new HashSet<>(extractRandomSubList(generalSkillList,
                                                                                     0.1, 0.3, 0.5, 0.7, 0.9, 1.0));
+            CovidRiskType covidRisk = extractRandomElement(Arrays.asList(CovidRiskType.values()));
```

Comment:
I believe that the distribution of covid risk type should follow a bell curve. As best as I can tell, the current distribution is uniform.